DependencyInjection/SncRedisExtension.php

```diff
@@ -276,7 +276,18 @@ protected function loadPhpredisClient(array $client, ContainerBuilder $container
         $phpredisDef->addArgument($client['alias']);
         $phpredisDef->addTag('snc_redis.client', array('alias' => $client['alias']));
         $phpredisDef->setPublic(false);
-        $phpredisDef->setLazy(true);
+
+        // Older version of phpredis extension do not support lazy loading
+        $minimumVersionForLazyLoading = '4.1.1';
+        $supportsLazyServices = version_compare($phpRedisVersion, $minimumVersionForLazyLoading, '>=');
+        if (!$supportsLazyServices) {
+            @trigger_error(
+                sprintf('Lazy loading Redis is not supported on PhpRedis %s. Please update to Phpredis %s or higher.', $phpRedisVersion, $minimumVersionForLazyLoading), 
+                E_USER_WARNING
+            );
+        } else {
```

Comment:
There's really no reason for an `else` here, this is basically saying

```php
else if ($var === true) {
    set(true);
}
```

which is equivalent to just

```php
set($var);
```

Reply:
@B-Galati up to you

Reply:
@dkarlovi You mean we could do `$phpredisDef->setLazy($supportsLazyServices);` instead of the else block right?
If yes it sounds better :+1:

Reply:
@B-Galati yes.